c10/macros/Export.h

```diff
@@ -1,11 +1,95 @@
 #ifndef C10_MACROS_EXPORT_H_
 #define C10_MACROS_EXPORT_H_
 
+/* Header file to define the common scaffolding for exported symbols.
```

Comment:
Changes in this file is just to revert my previous commit, because torch/standalone/macros/Export.h uses different macro namings now.